android/src/app/organicmaps/bookmarks/Holders.java

```diff
@@ -275,6 +276,19 @@ void setSize(@PluralsRes int phrase, int size)
       mSize.setText(mSize.getResources().getQuantityString(phrase, size, size));
     }
 
+    void setZeroSize()
+    {
+      setSize(R.plurals.objects, 0);
+    }
+
+    void setDetailedSize(int tracks, int bookmarks)
+    {
+      Resources resources = mSize.getResources();
+      String tracksSize = resources.getQuantityString(R.plurals.tracks, tracks, tracks);
+      String bookmarksSize = resources.getQuantityString(R.plurals.places, bookmarks, bookmarks);
+      mSize.setText(tracksSize + " + " + bookmarksSize);
```

Comment:
A new, single translatable string like "%d tracks, %d bookmarks" would be more flexible for different languages. See docs/TRANSLATIONS.md